### ISSUE TITLE
Summertime

### ISSUE THREAD

**Issue by elkiaer-dk** [piggz/harbour-amazfish#355]
Here after summertime data can not be download to phone .

**Comment by piggz** [comment]
Yeah, its an annoying problem ive never worked out! there is a workaround, in the settings, you can skip the "A" timer on an hour or 2 and it will start working
